README.md

```diff
@@ -52,6 +52,21 @@ Current Development (SNAPSHOT) downloads
   * JDK8: [oshi-core-6.8.4-SNAPSHOT](https://central.sonatype.com/service/rest/repository/browse/maven-snapshots/com/github/oshi/oshi-core/6.8.4-SNAPSHOT)
   * JPMS: [oshi-core-java11-6.8.4-SNAPSHOT](https://central.sonatype.com/service/rest/repository/browse/maven-snapshots/com/github/oshi/oshi-core-java11/6.8.4-SNAPSHOT/)
 
+OSHI Java 25+ Module
+----------------------------
+A new module, **`oshi-core-java25`**, is now available.
+
+- **Purpose:** This module intends to provide API-compatible implementations using the JDK Foreign Function & Memory (FFM) API, replacing JNA for native access over time with community contributions.
+- **Compatibility:**  
```

Comment:
Should mention a reduction in OS support (at least initially) until other operating systems have JDK25 options.